packages/imperative/src/imperative/src/config/cmd/edit/edit.definition.ts

```diff
@@ -24,8 +24,7 @@ export const editDefinition: ICommandDefinition = {
     summary: "edit config files",
     description: `Edit an existing config file in your system's default text editor.\n\n` +
         `In a graphical environment, the application associated with JSON files will be launched. ` +
-        `In a command-line environment, vi will be launched. To override the command-line editor used, specify it in ` +
-        `the ${ImperativeConfig.instance.loadedConfig.envVariablePrefix}_EDITOR environment variable.`,
```

Comment:
lint error - ImperativeConfig

Reply:
👍Also occurs in packages/imperative/src/utilities/src/ProcessUtils.ts